frontend/views/containers/settings/UserProfile.vue

```diff
@@ -129,19 +129,22 @@ import L from '@view-utils/translations.js'
 
 const url = helpers.regex('url', /^(https?|ftp):\/\/[^\s/$.?#].[^\s]*$/i)
 
+// TODO: rename this file to match the name below!
+
 export default {
   name: 'SettingsUserProfile',
   mixins: [validationMixin],
   components: {
     Avatar
   },
   data () {
+    const attrCopy = Object.assign({}, this.$store.getters.currentUserIdentityContract.attributes || {})
```

Comment:
simplified:

```
const attrCopy = this.$store.getters.currentUserIdentityContract.attributes || {};
```

Reply:
It's not a copy if we do it that way. We don't want these values to be directly modifiable, otherwise they can get out of sync from the rest of the users.

Reply:
Oh, right, didn't thought about the clone part. So, you can simply use destructuring:

```
const attrCopy = { ...this.$store.getters.currentUserIdentityContract.attributes };
```